internal/command/migrate_to_v2/migrate_to_v2.go

```diff
@@ -772,3 +790,17 @@ func determineAppConfigForMachines(ctx context.Context) (*appconfig.Config, erro
 	}
 	return cfg, nil
 }
+
+func determineVmSpecs(vmSize api.VMSize) (*api.MachineGuest, error) {
+
+	preset := strings.Replace(vmSize.Name, "dedicated-cpu", "performance", 1)
+
+	guest := &api.MachineGuest{}
+	err := guest.SetSize(preset)
+
+	if err != nil {
+		return nil, fmt.Errorf("nomad VM definition incompatible with machines API: %w", err)
+	}
```

Comment:
I think we also want to set `guest.MemoryMB = vmSize.MemoryMB` because that is something `fly scale vm --memory ` and `fly scale memory` commands allows to do.

Reply:
oh! at some point I refactored machineGuest stuff to use .SetSize and I totally forgot that memory wasn't in there. thanks!